web/src/components/storage/dasd/DASDTable.test.tsx

```diff
@@ -53,8 +74,54 @@ describe("DASDTable", () => {
     });
 
     it("renders those devices", () => {
-      installerRender(<DASDTable />);
+      installerRender(<DASDTable aria-label="DASDs table" />);
       screen.getByText("active");
     });
+
+    it("does not allow to perform any action if not selected any device", () => {
+      installerRender(<DASDTable aria-label="DASDs table" />);
```

Comment:
I think these `aria-labels` are not needed at all. Please, check them. `DASDTable` component receive none props.